localization/am.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<localizationPack name="Armenian" version="1.0">
+	<currencies>
+		<currency name="Armenian Dram" iso_code="AMD" iso_code_num="051" sign="֏" blank="1" conversion_rate="0.0021" format="2" decimals="0" />
+	</currencies>
+	<languages>
+		<language iso_code="hy" />
+	</languages>
+	<taxes>
+		<tax id="1" name="ԱԱՀ AM 20%" rate="20" />
+		<tax id="2" name="ԱԱՀ AM 16.67%" rate="16.67" />
```

Comment:
Isn't this rate the same as the previous 20%? I can read [here](http://www.ardani.net/armenia/armenia-taxation-vad.php): 'The general VAT rate is 20% of the turnover of taxable goods and services, which is equal to 16,67% of VAT-inclusive prices'.